### PR TITLE
Update tslint "extends" requirement

### PR DESCRIPTION
**Pull request by sandersn**
formerly `dtslint/dt.json`, now `@definitelytyped/dtslint/dt.json`.

Also update a couple of baselines that were missed in earlier PRs.

### CODE REVIEW

File: src/_tests/fixtures/55210/mutations.json [hunk 1]
```diff
@@ -7,5 +7,14 @@
...
       }
     }
+  },
```
Comment (sandersn) [inline src/_tests/fixtures/55210/mutations.json:10]: this and the change in result.json are the result of an earlier PR as far as I can tell.

File: src/_tests/fixtures/55741/derived.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "hasMergeConflict": false,
   "isFirstContribution": true,
   "tooManyFiles": false,
+  "hugeChange": false,
```
Comment (sandersn) [inline src/_tests/fixtures/55741/derived.json:12]: same, from an earlier PR I think

**Review by orta — APPROVED** [review]
Looks good to me

**Comment by elibarzilay** [comment]
(Bah.  I've had those changes in my directory, but I somehow missed adding them to that PR...)

**Pull request merged**